documentation/course/e2/README.md

```diff
@@ -2209,13 +2209,25 @@ TODO @tchojnacki
 
 TODO @tchojnacki
 
-## PU `TIC/??`
+## PU `TIC/01`, `TIC/02`, `TIC/03`, `TIC/04`, `TIC/05`, `TIC/06`
+
+![TIC/01](./images/sequence-diagram-clabbert-TIC01.drawio.svg)
+
+## PU `TIC/06`, `TIC/11`
+
+![TIC/06](./images/sequence-diagram-clabbert-TIC06.drawio.svg)
+
+## PU `TIC/07`, `TIC/09`
+
+![TIC/07](./images/sequence-diagram-clabbert-TIC07.drawio.svg)
```

Comment:
Please make the file names and image alt names consistent with what @mlodybercik proposed.